src/components/Board/Board.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { jest, it, beforeAll } from '@jest/globals';
+import { act, render, waitFor } from '@testing-library/react';
+import Board from './Board';
+import { setFail } from '../../api/boardsApi';
```

Comment:
`setFail` is not exported from the `boardsApi` but from the `__mocks__/boardsApi.js`. I don't know what this is working but it shouldn't, Webstorm actually has some suggestions that this cannot be found on that location, so maybe consider changing it to the correct one.

Reply:
Jest is auto-connecting this to mock object and mock object is boardApi so... its Jest thing